packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -649,7 +647,7 @@ export class IndexedDbPersistence implements Persistence {
       typeof this.document.addEventListener === 'function'
     ) {
       this.documentVisibilityHandler = () => {
-        this.queue.enqueueAndForget<DbOwner | void>(() => {
+        this.queue.enqueueAndForget(() => {
```

Comment:
Did the generic type annotation intentionally get removed?

Reply:
Yeah, it's always 'void' and can be inferred.